src/snippets/responsive-bg-image.liquid

```diff
@@ -0,0 +1,36 @@
+{% comment %}
+    It creates a style tag containing "background-image" properties to load the correct image depending on the resolution
+    Dependencies:
+    - Lazysizes plugin (https://github.com/aFarkas/lazysizes) which enable responsive image with faster load time and better performance.
+    - Lazysizes Responsive Images as a Service (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/rias) To load the correct image size with our CDN
+    - Lazysizes Bgset (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset) To use responsive images on background-image (CSS)
+
+    Accepts:
+    - image: {Object} Image object
+
+    Usage:
+    In your liquid template file, copy the following line
+
+    <div class="responsive-bg-image-wrapper">
```

Comment:
Yeah, we need a wrapper because the `<div>` has some styling with the "padding-top: xxx%" and the image has position absolute to prevent the unwanted "shift" motion.